backend/src/main/java/com/festago/application/MemberTicketService.java

```diff
@@ -34,6 +42,50 @@ public MemberTicketResponse findById(Long memberId, Long memberTicketId) {
     @Transactional(readOnly = true)
     public MemberTicketsResponse findAll(Long memberId) {
         List<MemberTicket> memberTickets = memberTicketRepository.findAllByOwnerId(memberId);
-        return MemberTicketsResponse.from(memberTickets);
+        return memberTickets.stream()
+            .sorted(comparing(memberTicket -> memberTicket.getStage().getStartTime()))
```

Comment:
공연을 조회하고, 시작 시간을 조회하는 것 보다, 티켓의 입장 시간을 기준으로 정렬하는 것은 어떤가요?

첫 번째 이유로, 공연을 조회할 때 SELECT 쿼리를 한 번 더 날려야 한다는 이유도 있고 (fetch join을 사용하더라도 join이 발생)

두 번째 이유로, 입장 시간과 무대 시작 시간이 크게 차이가 날 것 같진 않아서요! (정말 특별한 경우가 아니고서 공연의 시작 시간과 입장 시간이 크게 차이 나지 않으니까요!)
```suggestion
            .sorted(comparing(MemberTicket::getEntryTime))
```

Reply:
DTO에 축제 및 공연 내용이 함께 조회되어서 이 때 stage SELECT쿼리가 날아가나 DTO 변환시 날아가나 SELECT 횟수는 똑같을 것 같아요!!

Reply:
1번, 3번 중 고르면 될 것 같은데.. 입장 시간이 가까운 순서로 해주는게 좋아 보이네요

Reply:
전 3번에 한표! 영화관같은거 생각해도 예매 내역은 보통 영화상영시간보다 예매시간 기준으로 정렬되는 것 같아요

Reply:
어차피 무대시간과 입장시간이 큰 차이가 없기 때문에 입장 시간으로 정리해주는 것이 좀 더 좋아보인요!! 1번 하겠습니다.

Reply:
전 1번에 마음 갑니다. 유사 서비스를 챙겨본 건 아니지만, 예매는 일찍 했지만, 몇주뒤에 열릴 공연이 더 빨리 조회되면 애매할 거 같아요.